SQL statements/createIndexes.sql

```diff
@@ -0,0 +1,11 @@
+EXPLAIN SELECT FIRSTNAME, LASTNAME FROM READERS WHERE FIRSTNAME = 'JOHN';
+EXPLAIN SELECT * FROM BOOKS WHERE TITLE = 'The Stranger';
+
+CREATE INDEX IDX_READERS_NAME_LASTNAME ON READERS (FIRSTNAME, LASTNAME);
+CREATE INDEX IDX_BOOKS_TITLE ON BOOKS (TITLE);
+
+EXPLAIN SELECT FIRSTNAME, LASTNAME FROM READERS WHERE FIRSTNAME = 'JOHN';
+EXPLAIN SELECT * FROM BOOKS WHERE TITLE = 'The Stranger';
+
+DROP INDEX IDX_READERS_NAME_LASTNAME ON READERS;
```

Comment:
czemu usuwasz indeksy?

Reply:
Bo jakbym chciał jeszcze raz uruchomić skrypt, to wtedy nie zobaczę różnicy przy pierwszym "Explain", a potem dostanę błąd w miejscu tworzenia indeksów, że już istnieję. Ogólnie myślałem, że dobrze jest posprzątać pracę na koniec